content/_references/sailor-compatibility-matrix.md

```diff
@@ -12,18 +12,24 @@ are feature-compatible. If a feature is available in a given version, you can be
 sure that Sailors for both programming languages are supporting this feature. The
 following table shows the support of the platform features in Sailor versions:
 
-| Features | v2.0.x | v2.1.x | v2.2.x |
-| :--- | :--- | :--- | :--- |
-|  Request-reply |  Yes |  Yes | Yes  |
-|  Startup Hook |  Yes |  Yes | Yes  |
-|  Init Hook |  Yes |  Yes | Yes  |
-|  Passthrough |  - |  Yes | Yes  |
-|  Shutdown Hook |  - |  - | Yes  |
+| Features            | v2.0.x | v2.1.x | v2.2.x | 2.5.x | *2.6.x* |
+| :---                | :---   | :---   | :---   | : --- | : --- |
+| Request-reply       | Yes    | Yes    | Yes    | Yes   | Yes   |
+| Startup Hook        | Yes    | Yes    | Yes    | -     |       |
+| Init Hook           | Yes    | Yes    | Yes    | -     |       |
+| Passthrough         | -      | Yes    | Yes    | Yes   | Yes   |
+| Shutdown Hook       | -      | -      | Yes    | -     |       |
+| Bunyan Logger       | -      | -      | -      | Yes   | Yes   |
+| Message Size        | -      | -      | -      | Yes   | Yes   |
```

Comment:
I think message size was only introduced in v 2.6 while disable passthrough exists since v 2.5

Reply:
corrected